.github/workflows/ci.yml

```
@@ -0,0 +1,123 @@
+      - name: Set up Go
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+          
+      - name: Build
+        env:
+          GOOS: ${{ runner.os }}
+          GOARCH: ${{ matrix.arch }}
+        run: |
+          go build -v -o bin/flow-test-go-${{ runner.os }}-${{ matrix.arch }} ./cmd/flow-test-go
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)